src/components/specific/models/models-card/models-card-model-preview/ModelsCardModelPreview.vue

```diff
@@ -3,17 +3,26 @@
     <div class="models-card-model-preview__index" v-if="images.length > 1">
       {{ `${image.index} / ${images.length}` }}
     </div>
-
-    <BIMDataModelPreview
-      v-if="image && image.url"
-      :previewUrl="image.url"
-      :width="377"
-      :height="342"
-      backgroundColor="var(--color-silver)"
-    />
-    <div v-else class="models-card-model-preview__placeholder">
-      {{ $t("ModelsCardModelPreview.emptyPreviewPlaceholder") }}
-    </div>
+    <template v-if="image && image.url">
+      <BIMDataModelPreview
+        class="models-card-model-preview__viewer"
+        :type="image.type === MODEL_TYPE.IFC ? '3d' : '2d'"
+        :previewUrl="image.url"
+        :width="377"
+        :height="342"
+        backgroundColor="var(--color-silver)"
+      />
+      <BIMDataIcon
+        class="models-card-model-preview__index__preview-icon"
+        :name="MODEL_ICON[image.type]"
+        size="l"
+      />
+    </template>
+    <template v-else>
+      <div class="models-card-model-preview__placeholder">
+        <BIMDataIcon :name="MODEL_ICON[image.type]" customSize="150" />
```

Comment:
If this works, it means that image is never nullish, so `v-if="image..."` line 6 is useless.

=> may be replace by a simple `v-if="image.url"`